### ISSUE TITLE
Shadows on KSC runway from grass, and shadows from grass look terrible b/c of abrupt shadow cutoff

### ISSUE THREAD

**Issue by Oman395** [Gameslinx/Tessellation#117]
Pretty much as the title says. Running arch w/ kde and x. Mods:
```
B9PartSwitch
EVE
EVE-HR
Harmony2
Kopernicus
ModularFlightIntegrator
ModuleManager
RealChute
RealPlume
RealPlume-StockConfigs
Scatterer
Scatterer-config
Scatterer-sunflare
SmokeScreen
SRBWaterfallEffects
StockWaterfallEffects
TUFX
Waterfall
```
![ksp](https://user-images.githubusercontent.com/77183348/199149786-d6e8791e-5046-4d81-9b37-c872e06677f8.png)

**Comment by TeddyDD** [comment]
![screenshot1](https://user-images.githubusercontent.com/4397533/199847087-ada2d932-ab58-4f77-8744-c042b838341c.png)
Same issue here, let me know what kind of debug info would be useful.

**Comment by TeddyDD** [comment]
Workaround: you can disable scatter shadows in Parallax config

**Comment by Oman395** [comment]
yeah, that's pretty much what I've found. Personally I think the weird shadows on the KSC are worth it because without scatter shadows everything else looks weird, but that's just me-- ideally we could have both though lol. I've also observed shadows being cast on my plane when flying over trees-- it looks like the shadows aren't properly taking into account height.

**Comment by PMunch** [comment]
I have the same problem, but I'm only running Kopernicus and Parallax. Running Manjaro with i3.

**Comment by Arndorferd** [comment]
Having issue as well still.

**Comment by Arndorferd** [comment]
Seems like a linux version issue for me, works in proton. Could it be related too https://github.com/Gameslinx/Tessellation/issues/124

**Comment by TeddyDD** [comment]
Most likely. I switched to proton anyway due to much better performance.

**Comment by Arndorferd** [comment]
The fps does seem better but I didn't compare the two without mods. So Im not sure if its mods like this causing the reduction.

**Comment by codereptile** [comment]
And here we are, still with this problem =(

**Comment by kane-thornwyrd** [comment]
i'm afflicted too :/
![175149](https://github.com/user-attachments/assets/173a8ef5-fb8c-431e-9b2f-a13d054d158e)
(I have the Radeon RX 7900 XT)
Mod list:
Avalanche Plumes (AvalanchePlumes 1.0.4)
B9 Part Switch (B9PartSwitch v2.20.0)
Better Load Save Game Renewed (BetterLoadSaveGame 2.5.3.5)
BetterBurnTime (BetterBurnTime 1.10)
BetterTimeWarpContinued (BetterTimeWarpCont 2.3.13)
BLEAK - TUFX Profiles (BLEAK 1.2)
Breaking Ground (BreakingGround-DLC 1.7.1)
ClickThrough Blocker (ClickThroughBlocker 1:2.1.10.21)
Deferred (Deferred 1.2.5.0)
Environmental Visual Enhancements Redux (EnvironmentalVisualEnhancements 3:1.11.7.2)
EVE - Stock Planet Configs (EnvironmentalVisualEnhancements-HR 2:EVE-1.2.2-1)
Harmony 2 (Harmony2 2.2.1.0)
HUD Replacer (HUDReplacer 1.2.12-beta)
Kerbal Engineer Redux (KerbalEngineerRedux 1.1.9.5)
Kopernicus Planetary System Modifier (Kopernicus 2:release-1.12.1-214)
Making History (MakingHistory-DLC 1.12.1)
ModularFlightIntegrator (ModularFlightIntegrator 1.2.10.0)
Module Manager (ModuleManager 4.2.3)
Parallax (Parallax 2.0.8)
Parallax - Stock Planet Textures (Parallax-StockTextures 2.0.8)
Parallax - Stock Scatter Textures (Parallax-StockScatterTextures 2.0.8)
PlanetShine (PlanetShine 0.2.6.6)
PlanetShine - Default configuration (PlanetShine-Config-Default 0.2.6.6)
Scatterer (Scatterer 3:v0.0878)
Scatterer Default Config (Scatterer-config 3:v0.0878)
Scatterer Sunflare (Scatterer-sunflare 3:v0.0878)
Shabby (Shabby 0.4.0)
Shaddy (Shaddy v2.5)
SmokeScreen - Extended FX Plugin (SmokeScreen 2.8.14.0)
SpaceTux Library (SpaceTuxLibrary 0.0.8.6)
Stock Waterfall Effects (StockWaterfallEffects 0.8.0)
Toolbar (Toolbar 1:1.8.1.1)
Toolbar Controller (ToolbarController 1:0.1.9.11)
TUFX (TUFX 1.1.0)
Waterfall Core (Waterfall 0.10.3)
ZTheme (ZTheme v1.1.3)
